blivet/devices/lvm.py

```diff
@@ -1794,7 +1794,14 @@ def populate_ksdata(self, data):
 
 
 class LVMVDOPoolMixin(object):
```

Comment:
Does this class need a special `min_size`?

Reply:
I knew I forgot something important. I added a new commit setting `min_size` for vdo pools.